README.md

```diff
@@ -0,0 +1,258 @@
+# ServerAdvertisement3
+Very useful plugin for your server with which you can print messages to players. Messages are stored in file.
+
+## Features
+- Message Types:
```

Comment:
Also there is my newer Top Menu type per #16 (which just permits to adjust the `color` setting in the CFG):

[[](url)](https://user-images.githubusercontent.com/15228896/164123861-13dcb895-d5ae-4a77-b022-bf41b12f485c.jpg)